src/components/JobseekerProfileForm/profileUpload.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import ImageUploader from 'react-images-upload'
+import axios from 'axios'
+
+const ProfileUpload = () => {
+  const [state, setState] = React.useState({
+    images: [],
+  })
+  const onDrop = (image: any) => {
+    setState({ images: image })
+  }
+  //uploads images to backend
+  const onClickHandler = () => {
+    const data = new FormData()
+    data.append('file', state.images[0])
+    axios.post('/s3/image', data, {})
+  }
```

Comment:
This is wrong. You don't **POST** the `data` to server. You should **POST** the `fileName` and `fileType` instead. After that you will get back a signed url from aws and then **POST** `data` to the url you get back